crates/rpc/Cargo.toml

```diff
@@ -26,3 +26,4 @@ ssz_types.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
 warp.workspace = true
+tree_hash.workspace = true
```

Comment:
<img width="46" alt="image" src="https://github.com/user-attachments/assets/9797f5e6-f977-464e-9fbf-827db136ff11" />
a new line is missing